ocaml/runtime/caml/stack.h

```diff
@@ -104,6 +103,28 @@ typedef struct {
     num_debug is num_alloc if frame_size & 2, otherwise 1. */
 } frame_descr;
 
+typedef struct {
+  uintnat retaddr;
+  unsigned short marker;        /* frame_size_long */
```

Comment:
I hadn't spotted the alignment issue here!

The current version relies on the C compiler inserting two bytes of padding after `marker` to match what `Emitaux` does. I think this is how it works on every architecture, but it might be clearer to put an explicit `unsigned short _pad` after `marker`. (Or maybe this is more obscure. Not sure.)

Reply:
done